src/clj/rems/application/model.clj

```diff
@@ -257,6 +257,10 @@
   [application _event]
   application)
 
+(defmethod application-base-view :application.event/expiration-notifications-sent
+  [application _event]
+  application)
```

Comment:
We could record the time when they were sent to the model, but we don't seem to need it now, because we can show it from the event.